src/v/cluster/topics_frontend.h

```diff
@@ -64,6 +65,7 @@ class topics_frontend {
       config::binding<int16_t> minimum_topic_replication,
       config::binding<bool> partition_autobalancing_topic_aware);
 
+    using dispatch_to_leader = ss::bool_class<struct dispatch_to_leader_tag>;
```

Comment:
really small nit, but the way this typedef is introduced (in the same code block as `create_topics`) makes the reader think that it is somehow mostly related to `create_topics`, which is not the case.